File: src/main/webapp/app/entities/session/session-status.ts
/**
 * Enum for the SessionStatus
 */
export enum SessionStatus {
    CREATED = <any> 'CREATED',
    USERS_JOINING = <any> 'USERS_JOINING',
    ADDING_CARDS = <any> 'ADDING_CARDS',
    REVIEWING_CARDS = <any> 'REVIEWING_CARDS',
    CHOOSING_CARDS = <any> 'CHOOSING_CARDS',
    READY_TO_START = <any> 'READY_TO_START',
    IN_PROGRESS = <any> 'IN_PROGRESS',
    FINISHED = <any> 'FINISHED'
}

File: src/main/java/be/kdg/kandoe/backend/model/sessions/SessionStatus.java
package be.kdg.kandoe.backend.model.sessions;

/**
 * The possible statuses a session can have, in chronological order
 */
public enum SessionStatus {
    CREATED,
    USERS_JOINING,
    ADDING_CARDS,
    REVIEWING_CARDS,
    CHOOSING_CARDS,
    READY_TO_START,
    IN_PROGRESS,
    FINISHED
}
